#15 DataManager needs to be refactored - refactored

spase84 opened the pull request:
Refactoring DataManager to use Generic in get method regarding issue #15

spase84 committed:
#15 DataManager needs to be refactored - refactored

ppeelen commented:
Thanks for the PR, and nice update! It looks good however the tests are not uppdated.

spase84 committed:
Tests fixed;
Removed some spaces on empty strings at MentorCardViewController so linter don't show warnings now;

spase84 commented:
@ppeelen 

Tests updated now, thanks.
